srcpkgs/mono-addins/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'mono-addins'
+pkgname=mono-addins
+version=1.3
+revision=1
+wrksrc=${pkgname}-${pkgname}-${version}
+build_style=gnu-configure
+hostmakedepends="autoconf automake pkg-config"
+makedepends="mono-devel gtk-sharp2"
+short_desc="Generic framework for creating extensible applications"
+maintainer="KawaiiAmber <japaneselearning101@gmail.com>"
+license="MIT"
+homepage="https://www.mono-project.com/archived/monoaddins"
+distfiles="https://github.com/mono/${pkgname}/archive/${pkgname}-${version}.tar.gz"
+checksum=e2c4db9fd12087b795dd03a035c6ab8daffe157694d98dcfe227067913db5580
+
+do_configure() {
```

Comment:
Remove nugget file and just keep `./autogen.sh` in  `pre_configure()`

Reply:
Moved `./autogen.sh` to `pre_configure()`